### PR TITLE
feat(traffic_light_classifier): update classifier model

### PR DESCRIPTION
**Pull request by wep21**
## Description

- support model trained with [mmclassification](https://github.com/open-mmlab/mmclassification)
- add single image debug node
## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by wep21** [comment]
new model
```
ros2 run traffic_light_classifier single_image_inference --ros-args -p image_path:=/home/daisuke/Downloads/sample.jpg -p classifier_type:=1 -p label_file_path:=/home/daisuke/Downloads/lamp_labels.txt -p model_file_path:=/home/daisuke/Downloads/end2end.onnx
```
image

**Comment by wep21** [comment]
old model
```
ros2 run traffic_light_classifier single_image_inference --ros-args -p image_path:=/home/daisuke/Downloads/sample.jpg -p classifier_type:=1 -p label_file_path:=src/universe/autoware.universe/perception/traffic_light_classifier/data/lamp_labels.txt -p model_file_path:=src/universe/autoware.universe/perception/traffic_light_classifier/data/traffic_light_classifier_mobilenetv2.onnx -p input_name:=input_0 -p output_name:=output_0 -p apply_softmax:=true
```
image

File: perception/traffic_light_classifier/src/single_image_inference_node.cpp
Comment (yukkysaito) [inline perception/traffic_light_classifier/src/single_image_inference_node.cpp:65]: What is difference between this node and TrafficLightClassifierNodelet?

Comment (yukkysaito) [reply perception/traffic_light_classifier/src/single_image_inference_node.cpp:65]: This node is for debug?

Comment (wep21) [reply perception/traffic_light_classifier/src/single_image_inference_node.cpp:65]: Yes, this is just a debug node to test inference with a image file.

Comment (yukkysaito) [reply perception/traffic_light_classifier/src/single_image_inference_node.cpp:65]: Can you please write the word `debug` into the file or directory name and class name so that we can understand for debugging?

Comment (wep21) [reply perception/traffic_light_classifier/src/single_image_inference_node.cpp]: I addressed [here](https://github.com/autowarefoundation/autoware.universe/pull/2820/commits/b748a82bee5eb152a29dbb78305e94fb4e2b2b42)

**Review by yukkysaito — APPROVED** [review]
LGTM
please get approval from @xmfcx @kaancolak

**Review by ktro2828 — APPROVED** [review]

**Comment by wep21** [comment]
I added some fixes to support arrow signals.
image

**Comment by wep21** [comment]
@xmfcx @kaancolak friendly ping

File: perception/traffic_light_classifier/src/single_image_debug_inference_node.cpp
Comment (kaancolak) [inline perception/traffic_light_classifier/src/single_image_debug_inference_node.cpp:78]: I really love this debug tool but in order to use the CNN classifier, shouldn't we give the default label and onnx file as parameters?  these params don't declare in this node, and the default label file is just "label.txt".

File: perception/traffic_light_classifier/src/cnn_classifier.cpp
Comment (kaancolak) [inline perception/traffic_light_classifier/src/cnn_classifier.cpp:44]: Would it be helpful to also declare std and mean here? for people who use their custom-trained models

like this Pr : https://github.com/autowarefoundation/autoware.universe/pull/2794

**Review by kaancolak — COMMENTED** [review]
Hi Daisuke-san, sorry for the delay. I left some comments .

**Review by kaancolak — APPROVED** [review]

File: perception/traffic_light_classifier/src/cnn_classifier.cpp
Comment (wep21) [reply perception/traffic_light_classifier/src/cnn_classifier.cpp:44]: Yes, I think so. We can merge the PR after merging this PR.

Comment (kaancolak) [reply perception/traffic_light_classifier/src/cnn_classifier.cpp]: Also, ı would like to add, really loved this single inference debug tool, really useful for users, thanks for it  :)

**Pull request merged**
